proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -64,5 +64,8 @@ message QueryUpgradedConsensusStateRequest {
 // QueryUpgradedConsensusStateResponse is the response type for the Query/UpgradedConsensusState
 // RPC method.
 message QueryUpgradedConsensusStateResponse {
-  google.protobuf.Any upgraded_consensus_state = 1;
+  reserved 1;
+  reserved "option";
```

Comment:
```suggestion
```

You can remove this. This is to reserve a field name. Since you're using the same field name in the new field, there's no need to reserve anything.

Reply:
Good to know, thanks. I'm waiting on a confirmation from @AdityaSripal, but I think the direction with this pr might be to remove IBC entirely:

- create a duplicate proposal type in IBC that wraps an upgrade plan, then call ScheduleUpgrade from within IBC
- handle IBC related logic in IBC

This would make this field obsolete. Any preference for how to deprecate `QueryUpgradedConsensusStateResponse`? And for `Plan` I guess reserving the field and name would be sufficient

Reply:
Cool. Yes, reserving both field number and name should be enough.